x-pack/plugin/monitoring/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -1,2 +1,4 @@
 ALL-UNNAMED:
   - set_https_connection_properties # potentially required by apache.httpcomponents
+# Commented out for testing
```

Comment:
Any advice on how to gain confidence if this is required or not?

Reply:
That's a big question.
From experience, you need to try it on 8.x too (with JDK 17), and with FIPS. 
If that passes... I'd say we skip it. There is the risk we need to add some key at a later stage, if things start to fail. 
We have a JIRA about double-checking permissions/policies, maybe add a comment regarding this too?

Reply:
Can this be triggered using labels?

Reply:
I've gone through the permissions needed for the ldap sdk, this was even in the wrong place. The dependency is used by core and security